truenas_install/__main__.py

```diff
@@ -434,6 +466,21 @@ def main():
 
                 write_progress(0.5, "Performing post-install tasks")
 
+                for entry in TRUENAS_DATASETS:
+                    if not (force_mode := entry.get("mode")):
+                        continue
+
+                    os.chmod(f"{root}/{entry['name']}", force_mode)
+
+                # /usr will be readonly and so we want the ca-certificates directory to symlink
+                # to writeable location in /var/local
+                os.makedirs(f"{root}/usr/local/share", exist_ok=True)
+                shutil.rmtree(f"{root}/usr/local/share/ca-certificates", ignore_errors=True)
+                os.symlink("/var/local/ca-certificates", f"{root}/usr/local/share/ca-certificates")
+
+                with open(f"{root}/conf/truenas_root_ds.conf", "w") as f:
```

Comment:
Should we maybe have it named as `.json` instead ?

Reply:
Done